tests/codemods/sonar/test_sonar_sql_parameterization.py

```diff
@@ -1,14 +1,15 @@
 import json
 from pathlib import Path
 
-import pytest
-
 from codemodder.codemods.test import BaseSASTCodemodTest
 from core_codemods.sonar.sonar_sql_parameterization import SonarSQLParameterization
 
 SAMPLE_FILE_PATH = (
     Path(__file__).parents[2] / "samples" / "sonar" / "sql_parameterization.json"
 )
+SAMPLE_FILE_PATH_2 = (
```

Comment:
Just a tiny thing but I think I prefer the pattern of making `SAMPLE_FILE_PATHS = [ ... ]` instead of encoding indices in the name.